pkg/slo-controller/config/constants.go

```diff
@@ -33,5 +33,94 @@ Koordinator uses configmap to manage the configuration of SLO, the configmap is
    - <CPUBurstConfigKey>
 
 et.
-  TODO add a sample here
+
+For example, the configmap is as follows:
+
+```
+apiVersion: v1
+data:
+  colocation-config: |
+    {
+      "enable": true,
+      "metricAggregateDurationSeconds": 300,
+      "degradeTimeMinutes": 15,
+      "updateTimeThresholdSeconds": 300,
+      "resourceDiffThreshold": 0.1,
+      "nodeConfigs": [
+        {
+          "nodeSelector": {
+            "matchLabels": {
+              "kubernetes.io/kernel": "alios"
+            }
+          },
+          "updateTimeThresholdSeconds": 300,
+          "resourceDiffThreshold": 0.3
+        }
+      ]
+    }
+  cpu-burst-config: |
+    {
+      "clusterStrategy": {
+        "policy": "none",
+        "cpuBurstPercent": 1000
+      }
+    }
+  resource-qos-config: |
+    {
+      "nodeStrategies": [
+        {
+          "nodeSelector": {
+            "matchLabels": {
+              "kubernetes.io/kernel": "alios"
+            }
+          },
+          "lsClass": {
+            "memoryQOS": {
+              "enable": true,
+              "wmarkRatio": 50,
+              "wmarkScalePermill": 1000
+            }
+          },
+          "beClass": {
+            "memoryQOS": {
+              "enable": true,
+              "wmarkRatio": 90,
+              "wmarkScalePermill": 20
+            }
+          }
+        }
+      ]
+    }
+  resource-threshold-config: |
+    {
+      "clusterStrategy": {
+        "enable": true,
+        "cpuSuppressThresholdPercent": 75,
+        "cpuSuppressPolicy": "cpuBurst",
```

Comment:
the example is recommended to be valid. here the value of `cpuSuppressPolicy` may be `cpuset`